google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/changestream/ReadChangeStreamRetryTest.java

```diff
@@ -122,6 +122,15 @@ private StreamContinuationToken createStreamContinuationToken(@Nonnull String to
         .build();
   }
 
+  private StreamPartition createNewPartitionForCloseStream() {
```

Comment:
Is there a retry test that retries only the second partition? should it only retry the second partition?

Reply:
We don't need to worry about this because we don't retry after CloseStream: https://github.com/googleapis/java-bigtable/blob/main/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/changestream/ReadChangeStreamResumptionStrategy.java#L58

Reply:
But the comment also says: "will return an OK status right after sending a CloseStream.", but sounds like with this new change, server doesn't always return OK?

Reply:
Oh sorry for the confusion. The CloseStream.Status is different from the final status the server returns. If a CloseStream is returned, it's either one of the following:
1) CloseStream.Status==OK, which means the read finishes successfully, server will return an OK right after the CloseStream.
2) CloseStream.Status!=OK, CloseStream.NewPartitions/ContinuationTokens != empty, which means the client(i.e. beam) should start a new ReadChangeStream call with the new partitions. Server will still return an OK right after.

So in either case, server will return an OK right after CloseStream, and there's no need to retry after receiving a CloseStream.

I've updated the comment as well.